modules/configuring-a-pipeline-server.adoc

```diff
@@ -55,8 +55,31 @@ If you specify incorrect data connection settings, you cannot update these setti
 +
 [NOTE]
 ====
-The external database connection is secured by default. To specify an external database connection that is not secured with TLS, set the value of `database.customExtraParams.tls` to `false` in the `DataSciencePipelinesApplication` custom resource as shown here:
+The external database connection is secured by default. To specify an external database connection that is not secured with TLS, after creating the Pipeline Server, set the value of `database.customExtraParams.tls` to `false` in the `DataSciencePipelinesApplication` custom resource:
```

Comment:
You could change "after creating" to "after you create"

Reply:
Addressed in latest commit